lib-rgbpanel/include/rgbpanel.h

```diff
@@ -103,6 +116,8 @@ class RgbPanel {
 		uint8_t nBlue;
 	};
 	TColon *m_ptColons{nullptr};
+	uint8_t m_nFontID;
```

Comment:
m_FontID should be an enum class.

Reply:
Should it go here?

```
namespace rgbpanel {
static constexpr auto PWM_WIDTH = 84;
enum class TFontID { FONT_8x8, FONT_5x8 };
}  // namespace rgbpanel
```

Reply:
https://github.com/vanvught/rpidmx512/pull/159/commits/3e676a0ba901b61cd85ef3b8eec3fc49311eb9e4